Example/package.json

```diff
@@ -39,13 +39,16 @@
     "@react-navigation/core": "link:../react-navigation/packages/core/"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.0",
-    "@babel/preset-env": "^7.20.0",
-    "@babel/runtime": "^7.20.0",
-    "@react-native/babel-preset": "0.75.4",
-    "@react-native/eslint-config": "0.75.4",
-    "@react-native/metro-config": "0.75.4",
-    "@react-native/typescript-config": "0.75.4",
+    "@babel/core": "^7.25.2",
+    "@babel/preset-env": "^7.25.3",
+    "@babel/runtime": "^7.25.0",
+    "@react-native-community/cli": "15.0.0-alpha.2",
+    "@react-native-community/cli-platform-android": "15.0.0-alpha.2",
```

Comment:
🤔 Why do we need dependency for `rncli` here? 

I see it in diff, but that's interesting. 

Don't waste time on research here, I'm just posting open question if someone knows already.

Reply:
Just fyi, this is error I was having without it https://github.com/software-mansion/react-native-screens/actions/runs/11347709614/job/31559764615

Reply:
I see! Thanks for finding this out.

Reply:
https://github.com/software-mansion/react-native-screens/pull/2432 possibly related